src/clients/api/mutations/supply.ts

```diff
@@ -0,0 +1,15 @@
+import type { TransactionReceipt } from 'web3-core';
+import { Bep20 } from 'types/contracts';
+
+export interface ISupplyInput {
+  tokenContract: Bep20;
+  account: string | undefined;
+  amount: string;
+}
+
+export type SupplyOutput = void | TransactionReceipt;
+
+const supply = async ({ tokenContract, account, amount }: ISupplyInput): Promise<SupplyOutput> =>
+  tokenContract.methods.mint(amount).send({ from: account || undefined });
```

Comment:
`account` can only be a string or undefined, so no need to default it `undefined`.